packages/hardhat-core/src/internal/solidity/compiler/solc-info.ts

```diff
@@ -60,6 +60,7 @@ const defaultEvmTargets: { [key: string]: string } = {
   "0.8.21": "shanghai",
   "0.8.22": "shanghai",
   "0.8.23": "shanghai",
+  "0.8.24": "shanghai",
```

Comment:
shouldn't be `cancun`?

Reply:
nvm, I thought they'd upgrade the default evm version but it's still `shanghai`:

![image](https://github.com/NomicFoundation/hardhat/assets/1165424/11e0f418-75d5-43c0-b7b8-9eae84aadac5)